docs/source/changelog.rst

```diff
@@ -14,7 +14,29 @@ Changes in Jupyter Qt console
   to avoid a name clash with the `QWidget` properties. Note: the name change
   could be, in rare cases if a name collision exists, a code-breaking
   change.
-
+- add Shift+Tab shortcut to unindent text
+- add Ctrl+R shortcut to rename the current tab
+- add Alt+R shortcut to set the main window title
+- add Cmd+Alt+Left/Right shortcut to switch tabs on OSX
+- change Tab key behavior to always indent to the next increment of 4 spaces
+- change Home key behavior to alternate cursor between the beginning of text (ignoring leading spaces) and beginning of the line
+- add option to create new tab connected to the existing kernel
+- add support for PySide2
+- add support for Python 3.5
+- add support for 24bit ANSI color codes
+- improve documentation of various options and carified the docs in some places
+- move documentation to readthedocs
+- fix automatic indentation of new lines that are inserted in the middle of a cell
+- fix regression where prompt would never be shown for --existing consoles
+- fix "python.exe -m qtconsole" on windows
+- fix showing error messages when running a script using %run
+- fix "invalid cursor position" error and subsequent freezing of user input
+- fix syntax coloring when attaching to non-ipython kernels
+- fix printing when using QT5
+- fix Ctrl+K shortcut (delete until end of line) on OSX
+- fix history browsing (Up/Down keys) when lines are longer than the terminal width
+- fix saving HTML with inline PNG for Python 3 
+- various internal bugfixes 
```

Comment:
Please capitalize each new entry in this list, i.e.

`various internal ...` -> `Various internal ...`

etc.